schema/se/PrivateUnsecuredLoanApplicationCreated.yaml

```diff
@@ -0,0 +1,380 @@
+      housingType:
+        type: string
+        title: "Housing type for the applicant"
+        description: The form of housing in which the applicant resides
+        "meta:enum":
+          RENTED: The applicant rents his or her place of residence
+          OWN_APARTMENT: The applicant owns his or her apartment
+          OWN_HOUSE: The applicant owns his or her house
+          LIVE_IN: "the applicant doesn't directly rent or own a house but is rather a lodger"
+        enum:
+          - RENTED
+          - OWN_APARTMENT
+          - OWN_HOUSE
+          - LIVE_IN
+
+      housingCostPerMonth:
+        type: integer
+        title: 'Housing-related costs per month'
+        default: 0
+        minimum: 0
+        examples:
+        - 8000
+      employerName:
+        type: string
+        title: 'The company name of the employer'
+        examples:
+        - ACME AB
+      employerPhone:
+        type: string
+        pattern: '^\+[1-9][0-9]{1,14}$'
+        examples:
+        - "+465069158975"
+      childSupportReceivedMonthly:
+        type: integer
+        minimum: 0
+        title: "Money received for child-support each month"
+      childSupportPaidMonthly:
+        type: integer
+        minimum: 0
+        title: "Money paid for child-support each month"
+      maritalStatus:
+        type: string
+        title: "The marital status of the applicant"
+        enum:
+          - SINGLE
+          - MARRIED
+          - COHABITING
+      bankAccount:
+        title: "Applicant's bank account"
+        '$ref': '#/definitions/BankAccount'
+        "$linkVal": "BankAccount"
+      citizenships:
+        title: "Countries that the applicant is a citizen of"
+        '$ref': '#/definitions/countryCodeArray'
+        "$linkVal": "CountryCodeArray"
+      countriesOfResidence:
+        title: "Countries that the applicant is a resident of"
+        '$ref': '#/definitions/countryCodeArray'
+        "$linkVal": "CountryCodeArray"
+      taxResidentOf:
+        title: "Countries that the applicant is a resident of for tax-purposes"
+        '$ref': '#/definitions/countryCodeArray'
+        "$linkVal": "CountryCodeArray"
+      tentativeAddress:
+        title: "Address of the applicant for display purposes"
+        description:
+          "All actors are expected to perform proper due diligence in establishing the identity of the applicant"
+        '$ref': '#/definitions/Address'
+        "$linkVal": "Address"
+
+required:
+  - application
+  - dataProtectionContext
+properties:
+  broker:
+    type: "string"
+    title: 'Domain-name of the broker, in reverse order'
+    description: "Reversed DNS name for the broker, for example, example.com becomes com.example"
+    pattern: "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9-][a-zA-Z0-9]).)([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9-]*[A-Za-z0-9])"
+
+  brokerReference:
+    type: string
+    title: "Reference number for the broker"
+    description: "An arbitrary ID specified by the broker"
```

Comment:
Should we mention something about that it must be unique per application? and might not be reused?

Reply:
Yes